src/bin/enqueue-job.rs

```diff
@@ -13,7 +13,7 @@ fn main() -> Result<(), Error> {
     match &*job {
         "update_downloads" => Ok(tasks::update_downloads().enqueue(&conn)?),
         "dump_db" => {
-            let database_url = args.next().unwrap_or_else(|| env("DATABASE_URL"));
+            let database_url = args.next().unwrap_or_else(|| env("READ_ONLY_REPLICA_URL"));
```

Comment:
I assumed we are passing in the replica URL on the command line in production.

Could we add something like this to `.env.sample`, to make local testing easier?

    export READ_ONLY_REPLICA_URL=$DATABASE_URL

(I'm not sure variable expansion works in that file.)

Reply:
> I'm not sure variable expansion works in that file.

I think that does work, but I don't recall for certain.  In this case, I think I would prefer to remove the fallback entirely and add an error message suggesting to pass `$DATABASE_URL` via the shell.

Reply:
Sounds good.